lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.h

```diff
@@ -370,6 +371,20 @@ class DWARFASTParserClang : public lldb_private::plugin::dwarf::DWARFASTParser {
                          ParsedDWARFTypeAttributes &attrs);
   lldb::TypeSP ParseSubroutine(const lldb_private::plugin::dwarf::DWARFDIE &die,
                                const ParsedDWARFTypeAttributes &attrs);
+
+  bool
+  HandleObjCMethod(lldb_private::ObjCLanguage::MethodName const &objc_method,
+                   lldb_private::plugin::dwarf::DWARFDIE const &die,
+                   lldb_private::CompilerType clang_type,
+                   ParsedDWARFTypeAttributes const &attrs, bool is_variadic);
+
+  std::pair<bool, lldb::TypeSP>
+  HandleCXXMethod(lldb_private::plugin::dwarf::DWARFDIE const &die,
```

Comment:
> LinkCXXMethodDeclContextToDIE

Is that all the function does? It seems to be doing a bit more, so maybe something with the word `Parse` in it might be better.

I can't really say anything confidently, since I don't really understand this code, even though I've spend a lot of time staring at it last week.

Reply:
> Is that all the function does? It seems to be doing a bit more, so maybe something with the word Parse in it might be better.

True, could go for `ParseCXXMethod` I suppose

> I can't really say anything confidently, since I don't really understand this code, even though I've spend a lot of time staring at it last week.

Sadly having the same experience